Dockerfile

```diff
@@ -20,14 +20,11 @@ ENV PORT 3000
 COPY --link package.json yarn.lock .yarnrc.yml ./
 
 RUN set -eux; \
-	yarn && yarn cache clean
+	yarn && yarn playwright install --with-deps && yarn cache clean
 
 # copy sources
 COPY --link . ./
 
-RUN set -eux; \
```

Comment:
this change is unrelated, but it speeds up the builds and reduces the number of layers to store.